scripts/split-paired-reads.py

```diff
@@ -122,28 +123,28 @@ def main():
     screed_iter = screed.open(infile, parse_description=False)
 
     # walk through all the reads in broken-paired mode.
-    for index, is_pair, record1, record2 in broken_paired_reader(screed_iter):
-        if index % 100000 == 0 and index:
-            print >> sys.stderr, '...', index
-
+    paired_iter = broken_paired_reader(screed_iter)
```

Comment:
why are you making these changes? (record -> read, index -> n?) they seem unnecessary.